src/jit/x64/emit.dasc

```diff
@@ -1650,6 +1794,24 @@ void MVM_jit_emit_primitive(MVMThreadContext *tc, MVMJitCompiler *compiler, MVMJ
         |2:
         break;
     }
+    case MVM_OP_isinvokable: {
+        MVMint16 dst = ins->operands[0].reg.orig;
+        MVMint16 src = ins->operands[1].reg.orig;
+        | mov TMP1, aword WORK[src];
+        | mov TMP1, OBJECT:TMP1->st;
+        | mov TMP2d, dword STABLE:TMP1->invoke;
+        | cmp TMP2d, dword MVM_6model_invoke_default;
```

Comment:
`invoke` is a C function pointer and should be assumed to be 64 bits wide. This comparison of the lower 32 bits is a bug (that you'll probably get away with most of the time).

There's multiple correct ways of doing this; the simpler is to use `movabs` (or `mov64` in DynASM):

      mov64 TMP3, MVM_6model_invoke_default;
      cmp TMP3, aword STABLE:TMP1->invoke;

Reply:
egcc complained about casting a pointer to int when I tried this, but this works:
```
mov64 TMP2, ((uintptr_t)(MVM_6model_invoke_default));
cmp TMP2, STABLE:TMP1->invoke;
```

Reply:
This is how DynASM always does it, so it's correct